apps/examples/wc-vanilla/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "wc-vanilla",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "vite": "^4.3.0"
+  },
+  "dependencies": {
+    "@justeat/pie-design-tokens": "5.0.1",
+    "@justeattakeaway/pie-button": "0.10.0"
```

Comment:
I want all dependencies declared in this project to simulate a real example app. It shouldn't reference workspace dependencies at all.

Reply:
What is the benefit or necessity for doing so?

Reply:
It's simulating a real use case